### PR TITLE
Vstanciu

### PR DESCRIPTION
**Pull request by hobbitJS**

### CODE REVIEW

File: src/api/api.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import axios from 'axios'
```
Comment (gregurco) [inline src/api/api.js:1]: add new line after import

File: src/components/Layouts/Header.vue [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      <router-link :to="{ name: 'home' }">StackOverflow</router-link>
+    </v-toolbar-title>
+
+    <v-spacer/>
+
+    <div class="search">
+      <v-text-field
+        label="Search"
+        solo
+        @input="(q) => updateValue(q)"
```
Comment (gregurco) [inline src/components/Layouts/Header.vue]: you can write `@input="updateValue"` and it should work in the same way

File: src/components/Layouts/Header.vue [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      <v-btn flat>
+        <v-icon>help</v-icon>
+      </v-btn>
+    </v-toolbar-items>
+  </v-toolbar>
+</template>
+
+<script>
+import { mapActions, mapGetters } from "vuex";
+import SearchResult from "@/components/SearchResult";
```
Comment (gregurco) [inline src/components/Layouts/Header.vue]: add new line after imports block

File: src/components/Layouts/RightPart.vue [hunk 4]
```diff
@@ -0,0 +1,18 @@
+<template>
+  <div class="right-part">
+    <LastUserWidget />
+  </div>
+</template>
```
Comment (gregurco) [inline src/components/Layouts/RightPart.vue:5]: add new line after `</template>` and `<script>`

File: src/components/Layouts/RightPart.vue [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+<script>
+import LastUserWidget from '@/components/User/LastUserWidget'
+export default {
+  components: {
+    LastUserWidget
+  }
+}
+</script>
```
Comment (gregurco) [inline src/components/Layouts/RightPart.vue:15]: also new line here

File: src/components/Questions/List.vue [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    <div v-if="loading" class="text-xs-center mt-5">
+      <v-progress-circular
+        :width="1"
+        color="red"
+        :size="70"
+        indeterminate
+      />
+    </div>
+
+    <div v-for="question in questions" v-else :key="question.id">
```
Comment (gregurco) [inline src/components/Questions/List.vue]: it's not a good practice to keep `v-for` and `v-else` in same element. Split them in next mode:
```
<template v-else>
    <div v-for="question in questions" :key="question.id">
         // ...
    </div>
</template>
```

File: src/components/Questions/List.vue [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+                answers
+              </div>
+            </div>
+            <div class="mt-5 d-block text-xs-center">
+              {{ question.view_count }} views
+            </div>
+          </v-flex>
+          <v-flex xs10>
+            <div class="title">
+              <router-link :to="{name:'question', params: { id:question.id }}">
```
Comment (gregurco) [inline src/components/Questions/List.vue]: fix code style here

File: src/components/Questions/List.vue [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+      <!--      <hr>-->
+      <v-divider />
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapActions, mapGetters } from 'vuex'
+export default {
+  name: 'Home',
```
Comment (gregurco) [inline src/components/Questions/List.vue]: wrong name of component

**Review by gregurco — CHANGES_REQUESTED** [review]

**Comment by gregurco** [comment]
Too many problems with code style. Please run eslint and fix all problems.

File: src/api/api.js (hunk 1 above)
Comment (hobbitJS) [reply src/api/api.js:1]: fixed

File: src/components/Layouts/Header.vue (hunk 2 above)
Comment (hobbitJS) [reply src/components/Layouts/Header.vue]: fixed

File: src/components/Layouts/Header.vue (hunk 3 above)
Comment (hobbitJS) [reply src/components/Layouts/Header.vue]: fixed

File: src/components/Layouts/RightPart.vue (hunk 4 above)
Comment (hobbitJS) [reply src/components/Layouts/RightPart.vue:5]: fixed

File: src/components/Layouts/RightPart.vue (hunk 5 above)
Comment (hobbitJS) [reply src/components/Layouts/RightPart.vue:15]: fixed

File: src/components/Questions/List.vue (hunk 6 above)
Comment (hobbitJS) [reply src/components/Questions/List.vue]: fixed

File: src/components/Questions/List.vue (hunk 7 above)
Comment (hobbitJS) [reply src/components/Questions/List.vue]: fixed

File: src/components/Questions/List.vue (hunk 8 above)
Comment (hobbitJS) [reply src/components/Questions/List.vue]: fixed

**Comment by hobbitJS** [comment]
fixed

**Comment by gregurco** [comment]
@rapi could you please review?

**Review by rapi — COMMENTED** [review]
Copy from jfredz

**Pull request merged**